### DISCUSSION TITLE
Is useControl hook and Control component ready for prod?

### DISCUSSION THREAD

**Discussion by ejoc** [edmundhung/conform#653] [General]
I'm working on a form where two custom component (mobile, desktop) manage the same field. On the previous version (v0) I used  shadow input and refs. In v1 I'm using unstable_useControl hook and it is working well but I'm wondering if the useControl hook is ready for prod.

**Comment by edmundhung** [comment]
I am likely making some adjustment for the useControl hook and it will be a minor version bump if it happens. You can mark the version range to `~1.1.4` on your package.json to make sure only new patch version is accepted for now.

**Comment by m7mdBalboul** [comment]
Is the Control component ready for prod now?
